repo/packages/C/cassandra/1/command.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pip": [
+    "dcos<1.0",
+    "git+https://github.com/mesosphere/dcos-cassandra.git#dcos-cassandra=0.1.0"
+  ]
```

Comment:
Ideally we'll use whl for this, see https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/S/spark/1/command.json with an automated build, but this also works